src/routes/DigitalAssets/routes/Send/modules/digitalAssetsSend.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+
+export const OPEN_VIEW = '@@digitalAssetsSend/OPEN_VIEW'
+export const CLOSE_VIEW = '@@digitalAssetsSend/CLOSE_VIEW'
+
+export const SET_FIELD = '@@digitalAssetsSend/SET_FIELD'
+export const SET_INVALID_FIELD = '@@digitalAssetsSend/SET_INVALID_FIELD'
+
+export const CLEAN = '@@digitalAssetsSend/CLEAN'
+
+export type OpenViewParams = {|
+  +to?: string,
+  +asset?: string,
+  +txhash?: string,
+|}
+
+export function openView(params: OpenViewParams) {
+  return {
+    type: OPEN_VIEW,
+    payload: params,
+  }
+}
+
+export function closeView() {
+  return {
+    type: CLOSE_VIEW,
+  }
+}
+
+export function setField(fieldName: $Keys<DigitalAssetSendFormFields>, value: string) {
+  return {
+    type: SET_FIELD,
+    payload: {
+      fieldName,
+      value,
+    },
+  }
+}
+
+export function setFieldError(fieldName: $Keys<DigitalAssetSendFormFields>, message: string) {
+  return {
+    type: SET_INVALID_FIELD,
+    payload: {
+      fieldName,
+      message,
+    },
+  }
+}
+
+export function clearFieldError(fieldName: $Keys<DigitalAssetSendFormFields>) {
+  return setFieldError(fieldName, '')
+}
+
+export function clean() {
+  return {
+    type: CLEAN,
+  }
+}
+
+export type DigitalAssetSendAction =
+  ExtractReturn<typeof openView> |
+  ExtractReturn<typeof closeView> |
+  ExtractReturn<typeof setField> |
+  ExtractReturn<typeof setFieldError> |
+  ExtractReturn<typeof clean>
+
+// type TransactionPriorityType =
```

Comment:
I suggest to make map for TransactionPriorityType:
```
{
  'LOW': 1,
  'NORMAL': 1.5,
  'HIGH': 2,
}
```
where value by key is coefficient for current gasPrice

So, when we display tx data (on password screen) we can calculate fee: `coef * gasPrice * staticGasAmount`

Reply:
on form screen we only need to know coef
gasPrice could be requested from node on transition to password screen.